src/hotspot/share/opto/reg_split.cpp

```diff
@@ -1099,12 +1105,13 @@ uint PhaseChaitin::Split(uint maxlrg, ResourceArea* split_arena) {
                   Node *spill = new MachSpillCopyNode(MachSpillCopyNode::MemToReg, def, dmask, *tmp_rm);
                   insert_proj( b, insidx, spill, maxlrg );
```

Comment:
I think it missed `maxlrg` increment. May be also use `insidx++` here so in the following code you need to increment only by `delta`.

Reply:
added maxlrg++ and insidx++, the following insidx is added with delta